aws/main.tf

```diff
@@ -2,8 +2,7 @@
 
 
 module "redpanda-cluster" {
-  source                   = "redpanda-data/redpanda-cluster/aws"
```

Comment:
I would assume you'd want to change this back to an actual published module version prior to merging.

Reply:
yep!